### ISSUE TITLE
Unclickable links

### ISSUE THREAD

**Issue by soshimee** [Oharass/FreedomChat#12]
Links can't be clicked

**Comment by soshimee** [comment]
Yeah, I know this isn't a vanilla behavior, but I really need it for the server (it's very annoying when you have to manually type out all the links players send)

**Comment by soshimee** [comment]
No such plugins exist, since Bukkit already implements that.

**Comment by e-im** [comment]
CraftBukkit *did* implement that. It no longer does in 1.19.1+.

**Comment by e-im** [comment]
If you're still looking, I've made a plugin that implements this behaviour https://modrinth.com/plugin/clickablelinks (link may not work immediately, it is in their review queue).
